examples/advanced.html

```diff
@@ -8,6 +8,10 @@
     .examples {
       margin-top: 20px;
     }
+
+    media-loading-indicator {
+      --media-loading-icon-width: 100px;
```

Comment:
the default icon width of 44px came from having the media-loading-indicator in the control bar on first pass
see here: https://github.com/muxinc/media-chrome/pull/127/commits/cefd0337e6278549087668316022fe36dc03e962#diff-1e408dcff60df12596ea8ba2b3c22d7c01da0eb8f5bd6b9436d7804369fd5ec9R36

I especially reduced that size so it wouldn't expand the height of the control bar. 
https://github.com/muxinc/media-chrome/pull/127/commits/5ff7e82714b1528b530fbc10bd912219d92353ee

but yes in hindsight adding the loading indicator in the controlbar is probably less used.

Reply:
Ah, interesting. I don't hate having the loading spinner icon be default the same size as every other icon. But what we can do is add this CSS to the media-container within the centered-chrome scope, so that in the context of centered-chrome, the loading indicator has a default width of 100px. It wouldn't be the first place we've intentionally styled a specific element in a specific context to create a better default.